doc/gambit.txi

```diff
@@ -17985,7 +17985,319 @@ This situation can be avoided by having only one Scheme thread that
 does Scheme to C to Scheme calls.  Other Scheme threads are limited to
 simple Scheme to C calls that don't call back to Scheme.
 
-@node System limitations, Copyright and license, C-interface, Top
+
+@node Internal APIs, System limitations, C-interface, Top
+@chapter Internal APIs
+@cindex Internal
+
+Gambit has many internal functions that you might want to use
```

Comment:
"functions" -> "procedures" (that's the standard name for "functions" in Scheme)